schema/me/conversation.js

```diff
@@ -71,6 +72,7 @@ export const ConversationInitiatorType = new GraphQLObjectType({
       description: "The type of participant, e.g. Partner or User",
       type: new GraphQLNonNull(GraphQLString),
     },
+    initials: initials("name"),
```

Comment:
Hmm, nah we should be able to use `initials` under `me` below. Just like having `initials` part of the `Partner` schema, that seems like where it makes the most sense (`Me` rather than part of `Conversation`).

In terms of using it from the conversation, I think we can? Basically the query for conversations/messages already is under `me`:

```
me {
  id
  __id (or whatever)
  conversations {...}
}
```

so I think it'd just be as simple as putting `initials` there as well. Then, that can be referenced from the appropriate React components. Does that make sense?